lib/engines/content_object_store/test/factories/content_block_edition.rb

```diff
@@ -6,7 +6,7 @@
     schema { build(:content_block_schema) }
     creator
 
-    organisation { FactoryBot.create(:organisation) }
+    organisation { FactoryBot.build(:organisation) }
```

Comment:
oh nice idea, I remember this causing me a headache before

Reply:
Yeah, it took me a while to work out why the code was calling out to the Publishing API twice!